libs/core/src/lib/combobox/combobox.component.ts

```diff
@@ -534,6 +534,10 @@ export class ComboboxComponent
 
     /** @hidden */
     handleSearchTermChange(): void {
+        if (this.isEmptyValue) {
+            this._handleClearSearchTerm();
+        }
+
```

Comment:
before the regression the old behaviour was to set the value to `null` as soon as the label aka searchTerm is not an exact match to any items in the dropdown list.


https://user-images.githubusercontent.com/1498400/196356169-5881990b-e7a0-4f77-b025-5b0a16d5f4e1.mp4